.github/workflows/ci-libshortfin.yml

```
@@ -127,18 +151,24 @@ jobs:
         cmake --build build --target all
 
     - name: pip install shortfin
-      if: ${{ matrix.name != 'Ubuntu (Clang)(host-only)' && !contains(matrix.name, 'tracing') }}
+      if: ${{ matrix.name != 'Ubuntu (Clang)(host-only)' }}
       working-directory: ${{ env.LIBSHORTFIN_DIR }}
       run: |
         pip install -v -e build/
 
     - name: Test shortfin
-      if: ${{ matrix.name != 'Ubuntu (Clang)(host-only)' && !contains(matrix.name, 'tracing') }}
+      if: ${{ matrix.name != 'Ubuntu (Clang)(host-only)' && !matrix.tracing }}
       working-directory: ${{ env.LIBSHORTFIN_DIR }}
       run: |
         ctest --timeout 30 --output-on-failure --test-dir build
         pytest -s --durations=10 --timeout=30
 
+    - name: Test tracing shortfin
+      if: matrix.tracing
+      working-directory: ${{ env.LIBSHORTFIN_DIR }}
+      run: |
+        pytest -s --durations=10 --timeout=30 tests/invocation/mobilenet_tracing_test.py
```

How about these flags instead? There is only one test running, so we might as well get as much output from it as we can.

```suggestion
      run: |
        pytest \
          --durations=0 \
          --timeout=30 \
          --log-cli-level=info \
          tests/invocation/mobilenet_tracing_test.py
```

---

New logs look good:
https://github.com/nod-ai/shark-ai/actions/runs/13998942911/job/39200752810#step:12:34
```
tests/invocation/mobilenet_tracing_test.py::test_tracing_mobilenet 
-------------------------------- live log setup --------------------------------
INFO     root:binaries.py:185 Invoke IREE Tool: /opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/iree/compiler/tools/../_mlir_libs/iree-compile /tmp/pytest-of-runner/pytest-0/mobilenet_onnx0/mobilenet.mlir --iree-input-type=onnx --iree-vm-bytecode-module-output-format=flatbuffer-binary -o=/tmp/pytest-of-runner/pytest-0/mobilenet_onnx0/mobilenet_cpu.vmfb --mlir-print-debuginfo --mlir-print-op-on-diagnostic=false --iree-hal-target-device=llvm-cpu --iree-llvmcpu-target-cpu=host
-------------------------------- live log call ---------------------------------
INFO     tests.invocation.mobilenet_tracing_test:mobilenet_tracing_test.py:196 Tracy capture will be saved to: /tmp/pytest-of-runner/pytest-0/test_tracing_mobilenet0/capture.tracy
INFO     tests.invocation.mobilenet_tracing_test:mobilenet_tracing_test.py:198 Tracy capture will be using port 55139
INFO     tests.invocation.mobilenet_tracing_test:mobilenet_tracing_test.py:203 Tracy capture will be saved to: /tmp/pytest-of-runner/pytest-0/test_tracing_mobilenet0/capture.tracy
INFO     tests.invocation.mobilenet_tracing_test:mobilenet_tracing_test.py:219 Running MobileNet model with tracing enabled via subprocess
INFO     tests.invocation.mobilenet_tracing_test:mobilenet_tracing_test.py:135 Launching subprocess to run MobileNet model
INFO     tests.invocation.mobilenet_tracing_test:mobilenet_tracing_test.py:225 Waiting for tracy to finish collecting data...
INFO     tests.invocation.mobilenet_tracing_test:mobilenet_tracing_test.py:243 Tracy capture file size: 88402 bytes
PASSED                                                                   [100%]

============================== slowest durations ===============================
3.77s setup    tests/invocation/mobilenet_tracing_test.py::test_tracing_mobilenet
0.77s call     tests/invocation/mobilenet_tracing_test.py::test_tracing_mobilenet

(1 durations < 0.005s hidden.  Use -vv to show these durations.)
============================== 1 passed in 4.56s ===============================
```